mora/service/orgunit.py

```diff
@@ -205,14 +205,13 @@ def get_one_orgunit(c, unitid, unit=None,
     elif details is UnitDetails.FULL:
         unittype = common.get_uuid(rels['enhedstype'][0], required=False)
 
-        if rels['overordnet'][0]['uuid']is not None:
-            parent = get_one_orgunit(c, rels['overordnet'][0]['uuid'],
-                                     details=UnitDetails.FULL)
-            if parent is not None:
-                path = parent['path'] + '/' + parent['name']
-            try:
-                r['path'] = path
-            except NameError:
+        if rels['overordnet'][0]['uuid'] is not None:
+            r['parent'] = get_one_orgunit(c, rels['overordnet'][0]['uuid'],
+                                          details=UnitDetails.FULL)
+            if r['parent'] is not None:
+                slash = '' if not r['parent']['path'] else '/'
+                r['path'] = r['parent']['path'] + slash + r['parent']['name']
```

Comment:
If you include the current item in the path, you can simplify these two lines to simply be:

```python
 r['path'] = '/'.join((r['parent']['path'], r['name']))
```